database.go

```diff
@@ -990,6 +1018,17 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropSeries will delete all data with the seriesID
```

Comment:
no, retention policy is one of the right places to have it. If you're dropping a series, you should go from database -> retention policy -> shard group -> shard.